src/modules/ie-objects/components/NamesList/NamesList.tsx

```diff
@@ -0,0 +1,149 @@
+import { Button, TextInput } from '@meemoo/react-components';
+import clsx from 'clsx';
+import React, {
+	type ChangeEvent,
+	type FC,
+	type KeyboardEvent,
+	useCallback,
+	useEffect,
+	useState,
+} from 'react';
+import PerfectScrollbar from 'react-perfect-scrollbar';
+
+import { ConfidenceIndicator } from '@ie-objects/components/ConfidenceIndicator/ConfidenceIndicator';
+import { type NamesListProps } from '@ie-objects/components/NamesList/NamesList.types';
+import { Icon } from '@shared/components/Icon';
+import { IconNamesLight } from '@shared/components/Icon/Icon.enums';
+import { tText } from '@shared/helpers/translate';
+
+import styles from './NamesList.module.scss';
+import 'react-perfect-scrollbar/dist/css/styles.css';
+
+export const NamesList: FC<NamesListProps> = ({ className, names, onZoomToLocation }) => {
+	const [searchTermsTemp, setSearchTermsTemp] = useState('');
+	const [searchTerms, setSearchTerms] = useState('');
+	const [filteredNames, setFilteredNames] = useState(names);
+
+	function handleOnChange(evt: ChangeEvent<HTMLInputElement>): void {
```

Comment:
Is there a reason why you write functions with `function` and not with an arrow function?

Reply:
nee, webstorm genereerd ze zo als je kiest voor: generate function in component scope
ik pas het aan 👍